config/locales/models/activity.en.yml

```diff
@@ -46,7 +46,6 @@ en:
         planned_end_date: For example, 28 11 2020
         planned_start_date: For example, 27 3 2020
         sector_category: What area of the economy or society is your %{level} helping? For example, research, education or small to medium-sized enterprise (SME) development. Choose one of the <a href='https://www.oecd.org/dac/stats/documentupload/2015%20CRS%20purpose%20codes%20EN_updated%20April%202016.pdf' target='_blank' class='govuk-link'>CRS purpose codes</a>
-        status: This is the stage your %{level} is currently at
```

Comment:
@duffylouise Did you intend to remove this hint? It is needed in the page

Reply:
The content review recommended that the title and hint text be merged so that the title is now a question which incorporates the hint text. Does that make sense?

Reply:
It does, we just need to remove the hint from the view template (I can do that)

Reply:
Thank you!